datalad_next/create_sibling_webdav.py

```diff
@@ -327,7 +330,7 @@ def _dummy(ds, refds, **kwargs):
             else:
                 dsurl = url
 
-            return _create_sibling_webdav(
+            yield from _create_sibling_webdav(
```

Comment:
I believe that this is not actually necessary, because it is OK to "return" a generator, and still have it be a generator. I used `return`, because `foreach-dataset` takes whatever is returned and places it into the result property. It neither expects a generator, nor does it have any handling for such a case. I consider this semantically more appropriate by a small margin.

Reply:
Can see that. On the other hand: That distinction implies that the use of return vs yield needs to be reevaluated with every change in implementation. So - not sure. No too strong opinion.

Reply:
Turns out to be false. Would not work with `**` unless yielded.

Reply:
Done